apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/block-preview.js

```diff
@@ -11,18 +11,24 @@
  */
 import { BlockEditorProvider, BlockList } from '@wordpress/block-editor';
 import { Disabled } from '@wordpress/components';
+import { PostTitle } from '@wordpress/editor';
 
 // Exists as a pass through component to simplify automatted testing of
 // components which need to `BlockEditorProvider`. Setting up JSDom to handle
 // and mock the entire Block Editor isn't useful and is difficult for testing.
 // Therefore this component exists to simplify mocking out the Block Editor
 // when under test conditions.
 export default function( { blocks, settings, recomputeBlockListKey } ) {
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	return (
 		<BlockEditorProvider value={ blocks } settings={ settings }>
 			<Disabled key={ recomputeBlockListKey }>
+				<div className="template-title">
```

Comment:
Do we need this extra `div`? It appears to work without it.

Reply:
I've used this class to add some padding to the title element. I think it's ok.